test/lib/viewers/ResultsViewer.test.ts

```diff
@@ -78,56 +79,20 @@ describe('ResultsViewer implementations', () => {
 			viewer.view(results);
 
 			// ==== ASSERTIONS ====
-			// Assert against our messages.
-			const expectedDisplayEvents: DisplayEvent[] = [{
-				type: DisplayEventType.LOG,
-				data: getMessage(BundleName.ResultsViewer, 'summary.found-results', [DETAIL_RESULTS_CUTOFF, 1])
-			}];
-			expectedDisplayEvents.push(...repeatDetailExpectation(rule1, engine1, violations[0], DETAIL_RESULTS_CUTOFF));
-			expectedDisplayEvents.push(...createBreakdownExpectations(DETAIL_RESULTS_CUTOFF, [{
-				sev: rule1.severityLevel,
-				count: DETAIL_RESULTS_CUTOFF
-			}]));
+			// Compare the text in the events with the text in our comparison file.
+			// NOTE: If you actually look at the comparison file, it will have weird control characters in it to match
+			//       the styling applied to the output. If you need to regenerate the file, you cannot simply copy-paste,
+			//       because that removes the styling. Instead, you should use fs.writeFileSync to write directly to the
+			//       comparison file.
```

Comment:
See my other comment... which suggests we actually only have a few tests that keep those control characters in it (i.e. just the StylingUtil tests). Everywhere else, i say we always strip the styling and compare against plain text.

Reply:
Yup.